src/pages/api/auth/[...nextauth].ts

```diff
@@ -21,4 +21,23 @@ export default NextAuth({
   pages: {
     signIn: '/auth/signin',
   },
+  session: {
+    strategy: 'jwt',
+  },
+  callbacks: {
+    async session({ session, token, user }) {
```

Comment:
idt we need a session callback since we don't use the user's id or accesstoken anywhere on the client.